### ISSUE TITLE
JSON exception on the pages - After completing Chapter 1

### ISSUE THREAD

**Issue by sambasivaraja** [Adobe-Marketing-Cloud/aem-guides-wknd-events#19]
Am trying to integrate AEM and React SPA editor based on the steps provided in the Chapter 1

Chapter 0 - Able to complete all primary development setup and able to see the desired result in AEM local

Chapter 1 - Have followed the same steps and deployed the code to AEM 6.4 local . When I tried to open the sample page am getting the error as below .

I have also tried with installing the same package they have provided , but still the same issue.

react-app.js:1 Uncaught (in promise) SyntaxError: Unexpected token < in JSON at position 0

    at react-app.js:1
Am not able to complete this chapter because of this issue . Can anyone advise on this

**Comment by godanny86** [comment]
Hi @sambasivaraja please ensure that you are running AEM 6.4 with Service Pack 2 installed. See https://github.com/Adobe-Marketing-Cloud/aem-guides-wknd-events/issues/17 for more details.

**Comment by mprabagar80** [comment]
I am also getting the same error and I do have AEM 6.4 with SP 2 installed.

**Comment by godanny86** [comment]
Hi @mprabagar80 
What happens when you view /content/wknd-events/react.model.json in the browser? Does it render the json as expected or is an error thrown? Also do you see anything in the AEM server error.log?

**Comment by sambasivaraja** [comment]
@godanny86  - 

The cookie is not passed in the AJAX request[On the page load], so i get redirected to the login page.
This prevents the site being properly loaded.

Below you find the headers.

General:

Request URL: http://localhost:4502/content/wknd-events/react.model.json
Request Method: GET
Status Code: 302 Found
Remote Address: [::1]:4502
Referrer Policy: no-referrer-when-downgrade
Request Header:

Accept: */*
Accept-Encoding: gzip, deflate, br
Accept-Language: de-DE,de;q=0.9,en-US;q=0.8,en;q=0.7
Cache-Control: no-cache
Connection: keep-alive
Host: localhost:4502
Pragma: no-cache
Referer: http://localhost:4502/content/wknd-events/react.html
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.117 Safari/537.36
Response Header:

HTTP/1.1 302 Found Date: Wed, 17 Oct 2018 11:43:27 GMT Location: http://localhost:4502/libs/granite/core/content/login.html?resource=%2Fcontent%2Fwknd-events%2Freact.model.json&$$login$$=%24%24login%24%24&j_reason=unknown&j_reason_code=unk nown Content-Length: 0

NOTE : When I open the directly in browser /content/wknd-events/react.model.json it gives the desired JSON result . Issue is mainly around the AJAX that happen on Page load

**Comment by godanny86** [comment]
Thanks @sambasivaraja for the extra information. Forgive me if you have already tried this but do you get the same behavior with the [We.Retail Journal App](https://github.com/adobe/aem-sample-we-retail-journal) ?

Also is this behavior consistent across browsers? IE, Firefox, safari..?

**Comment by sambasivaraja** [comment]
> Thanks @sambasivaraja for the extra information. Forgive me if you have already tried this but do you get the same behavior with the [We.Retail Journal App](https://github.com/adobe/aem-sample-we-retail-journal) ?
> 
> Also is this behavior consistent across browsers? IE, Firefox, safari..?

Yes @godanny86  . I get the same behaviour with Journal too .

The same issue happening for all browser . _**{ Just thinking loud is there any configuration you have enable in your AEM Author local to skip authentication }_**

I could see similar issue has been reported from couple of Users .Just found few related items in adobe forum too

**Comment by pfauchere** [comment]
Here is some more info:

Yes, it seems to be an authorization issue where the `Authorization` header is expected.

On an authoring instance and with the current tutorial or the sample content, you may have to log in first using the AEM form.

If you are trying to access the model from a local development server, please refer to the latest version of the documentation [0].

It has also been reported that some browsers require a default authentication value (Edge, Chrome+Linux). We are investigating that issue.

[0] https://github.com/adobe/aem-sample-we-retail-journal/blob/master/react-app/DEVELOPMENT.md#gotchas---you-should-read-this

**Comment by sambasivaraja** [comment]
Thanks @pfauchere  : Can you guide me how to login via AEM forms .. Please share sample OOTB page URL that you are referring too

**Comment by mprabagar80** [comment]
I found the problem with this issue. It is to do with Chrome 67. Chrome v67  is not sending the cookies  while calling http://localhost:4502/content/wknd-events/react.model.json.  When I checked from another browser with Chrome v70, it worked.  Then I realized in my laptop, Chrome updates were disabled by admin. I just had to go to regedit, make the chrome auto update, then update Chrome and now all is good.
